src/components/Home/container.js

```diff
@@ -109,7 +109,9 @@ export default Container => class extends Component {
             },
           },
           startingDate: item.start === date,
+          startDate: item.start,
           endingDate: item.end === date,
+          endDate: item.end,
```

Comment:
The naming from lines 111-114 is a little confusing and might be worth changing, but I'm still going to approve to let you keep moving forward.